src/FrozenClock.php

```diff
@@ -22,6 +22,11 @@ public function setTo(DateTimeImmutable $now): void
         $this->now = $now;
     }
 
+    public function modify(string $modify): void
+    {
+        $this->now = $this->now->modify($modify);
```

Comment:
According to the signature, `DateTimeImmutable#modify()` can return `false` (PHP <= 8.3).
Shouldn't we cover that with tests?

More info: https://github.com/php/php-src/commit/494615fcb8c1fb5984e0e7d666e51a2dfc6bee55